python-client/tests/fixtures/german_credit_scoring.py

```diff
@@ -142,3 +144,18 @@ def german_credit_model(german_credit_data) -> GiskardModel:
         classification_threshold=0.5,
         classification_labels=clf.classes_,
     )
+
+
+@pytest.fixture()
+def german_credit_always_default_model() -> GiskardModel:
+    X = np.array([0])
+    y = np.array(["Default"])
+
+    dummy = DummyClassifier(strategy="constant", constant=np.array(["Default"]))
+    dummy.fit(X, y)
+
+    return GiskardModel(
+        prediction_function=dummy.predict,
+        model_type='regression',
```

Comment:
this one is a 'classification' model


actually to build a dummy model that would behave simmilar to a `german_credit_model` but return 'Default' all the time you can just do more or less the same as the `german_credit_model` does including the X_train, Y_train creation:


```
@pytest.fixture()
def german_credit_always_default_model(german_credit_data) -> GiskardModel:
    Y = german_credit_data.df["default"]
    X = german_credit_data.df.drop(columns="default")

    dummy = DummyClassifier(strategy="constant", constant="Default")
    dummy.fit(X, Y)

    return GiskardModel(
        prediction_function=dummy.predict_proba,
        model_type="classification",
        feature_names=list(input_types),
        classification_threshold=0.5,
        classification_labels=dummy.classes_,
    )

```

Like this your dummy model will be exactly like a "normal" one except that it'll return Default label all the time